packages/integration-testsuite/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Maybe worth a comment in this file or elsewhere noting that we don't dual-build the integration test suite. (Why? I guess the advantage of ESM is that bundlers can be smarter, and nobody is bundling the test suite into their apps. Also maybe some of our test deps don't support ESM? Worth noting though.0